src/components/ErrorComponent/ErrorComponent.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React, { FC, useEffect } from 'react';
+
+interface Props {
+  error: string;
+  onChangeError: (error: string) => void;
+}
+
+export const ErrorComponent: FC<Props> = React.memo(({
+  error,
+  onChangeError,
+}) => {
+  useEffect(() => {
```

Comment:
```suggestion
}) => {

  useEffect(() => {
```

Reply:
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/123888109/1ce2d312-2172-4847-b15a-b24d4f502874)